spec.html

```diff
@@ -26279,7 +26296,10 @@ <h1>Function.prototype.toString ( )</h1>
         <p>When the `toString` method is called, the following steps are taken:</p>
         <emu-alg>
           1. Let _func_ be the *this* value.
-          1. If _func_ is a <emu-xref href="#sec-built-in-function-objects">built-in function object</emu-xref>, return an implementation-defined String source code representation of _func_. The representation must have the syntax of a |NativeFunction|. Additionally, if _func_ has an [[InitialName]] internal slot and _func_.[[InitialName]] is a String, the portion of the returned String that would be matched by |NativeFunctionAccessor?| |PropertyName| must be the value of _func_.[[InitialName]].
+          1. If _func_ is a <emu-xref href="#sec-built-in-function-objects">built-in function object</emu-xref>, then
```

Comment:
Rather than this `If`, I think it would be simpler to just switch the order of steps 2 and 3 in this algorithm, so that the `[[SourceText]]` check happens before the built-in function check.